engine/crates/integration-tests/tests/federation/basic/variables.rs

```diff
@@ -192,32 +192,32 @@ fn list_coercion() {
 fn invalid_ints() {
     insta::assert_json_snapshot!(error_test("int", "Int!", 1.5), @r###"
     [
-      "Variable $input got an invalid value: found a Float value where we expected a Int scalar at $input"
+      "Variable $input has an invalid value. Found value 1.5 which cannot be coerced into a Int scalar"
```

Comment:
```suggestion
      "Variable $input has an invalid value. Found value 1.5 which cannot be coerced into an Int scalar"
```

et.al. for the rest... Annoying yes, but it's a public error :)

Reply:
yeah well, giving up on this one. Too annoying for me and tons of places where this would be needed even with enum names.